CMSgov/cms-carts-seds
# Mdct 393 - Confirmation Dialog Added

# Description
A dialog was required to review and/or submit the report. The user is prompted with two options when trying to submit.

![Screen Shot 2022-07-05 at 12 44 50 PM](https://user-images.githubusercontent.com/55826377/177403842-8fd201c4-8bd2-4ff4-8879-791265922d2b.png)


https://qmacbis.atlassian.net/browse/MDCT-393

## How to test
Run locally and try to submit, you should be prompted to double check or to submit

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
